cmd/mimir/config-descriptor.json

```
@@ -2306,12 +2306,12 @@
         },
         {
           "kind": "field",
-          "name": "exemplars_update_period",
+          "name": "tsdb_config_update_period",
           "required": false,
-          "desc": "Period with which to update per-tenant max exemplar limit.",
+          "desc": "Period with which to update per-tenant TSDB config.",
```

```suggestion
          "desc": "Period with which to update per-tenant TSDB configuration.",
```
Change "config" to "configuration" in most places to conform to style; in general, spell out words.